hacspec/tests/test_integers.rs

```diff
@@ -0,0 +1,190 @@
+#![allow(non_snake_case)]
+
```

Comment:
Wouahou, thanks for all the testing !

Reply:
:slightly_smiling_face: I think we need way more. But as I always need to implement some stuff to actually make the tests pass adding more tests always takes a lot of time :grimacing: